### DISCUSSION TITLE
Android 11-Even if "autoGrantPermissions" desired capability is set "true" location alert comes and it blocks automation

### DISCUSSION THREAD

**Discussion by way2singh** [appium/appium#15212] [Q&A]
Hi ,

I am facing issue in Android 11, even if  "autoGrantPermissions" desired capability is set "true" location alert comes and it blocks automation. Till Android 10 this capability was working fine but recently i have switched to Android 11. I tried changing appium version 1.17.0,1.18.0,1.19.1,1.20.2 but still issue is there. i even tried fullReset capabilty .Please suggest if someone has faced and applied any solution to it.

**Comment by way2singh** [comment]
![Capture](https://user-images.githubusercontent.com/64783403/112126808-d2318180-8bea-11eb-93da-c5cb17c0211a.PNG)

**Comment by KazuCocoa** [comment]
https://github.com/appium/appium/issues/15214
